ddp-workspace/projects/ddp-sdk/src/lib/services/irbPassword.service.ts

```diff
@@ -34,7 +34,7 @@ export class IrbPasswordService extends NotAuthenticatedServiceAgent<boolean> {
                 }
                 return loggedIn;
             }),
-            catchError(this.handleError));
+            catchError(this.handleError.bind(this)));
```

Comment:
We pass `this.handleError` as a callback, so, we have to save context, to make this.log work properly.
Without this, we will get an error in the console if the backend returns an error:
![image](https://user-images.githubusercontent.com/30126907/102803735-c9ea6000-43c9-11eb-927e-ff87880c7622.png)